truss/build.py

```diff
@@ -274,6 +285,17 @@ def mk_truss(
     )
 
 
+def mk_truss(**kwargs):
+    """DEPRECATED, use truss.create() instead
+
+    Args:
+        model (Any): An in-memory model object
+    Returns:
+        TrussHandle
+    """
+    return create(kwargs)
```

Comment:
Should we log here a deprecation warning here?

Reply:
Yes we probably should! Is there a standard way of doing that or do I just say `print("don't use this pls it is deprecated")`

Reply:
generally `logger.warn("plz no")`